tests/format/html/front-matter/__snapshots__/format.test.js.snap

```diff
@@ -116,3 +116,25 @@ Test <a href="https://prettier.io">abc</a>.
 
 ================================================================================
 `;
+
+exports[`unicode.html format 1`] = `
+====================================options=====================================
+parsers: ["html"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+---
+title: "ABC 漢字 🇯🇵"
```

Comment:
🇯🇵 consists of two Unicode code points, and each of them is represented as a surrogate pair in UTF-16.
Solutions like `[...input].map(ch => ch === "\n" ? "\n" : " ".repeat(ch.length)).join("")` doesn't pass 🇯🇵.